code/game/objects/items/storage/boxes/service_boxes.dm

```diff
@@ -216,7 +220,7 @@
 	. = ..()
 	atom_storage.max_slots = 24
 	atom_storage.set_holdable(list(/obj/item/toy/balloon/long))
-	atom_storage.max_total_storage = 24
+	atom_storage.max_total_storage = 72
```

Comment:
This is bad. Now people can fit 24 small items:

```
/// Usually items smaller then a human hand, (e.g. playing cards, lighter, scalpel, coins/holochips)
#define WEIGHT_CLASS_TINY 1
/// Pockets can hold small and tiny items, (e.g. flashlight, multitool, grenades, GPS device)
#define WEIGHT_CLASS_SMALL 2
```

Reply:
Ah, for the super high storage/slots items I specifically set them to only hold the item they spawn with and those items only. with the set_holdable() function, I figured that being able to carry 24 balloons is fine power-wise. and they can always decon and make normal boxes with it

Reply:
Good points.

Reply:
atom_storage.set_holdable(list(/obj/item/toy/balloon/long))
	
ist right there man

Reply:
there is a var that lets you insert objects regardless of size, which is what I thought was happening. Instead this one disables all objects from being inserted into the box.. which is kinda meh